src/components/Media/index.js

```diff
@@ -148,17 +153,18 @@ export default class Media extends PureComponent {
       <div
         {...compose(
           theme.placeholder,
+          !this.isLoaded && theme['placeholder--loading'],
```

Comment:
there are other icons, which will get blurred: Loading | Network error | 404 error | Offline

https://github.com/stereobooster/react-ideal-image/blob/master/introduction.md

Reply:
True, thanks for pointing it out. Need to think a bit about it.